src/components/DeckBuilderDryRunView/index.js

```diff
@@ -13,7 +14,15 @@ export default props => {
   const [mode, setMode] = React.useState(params.get('mode') || 'AUTOMATIC')
   const [modifier, setModifier] = React.useState('NONE')
   const [equalsMode, setEqualsMode] = React.useState(false)
-  const deck = modifyDeck(props.deck, modifier, equalsMode)
+  const addIdx = card => ({ ...card, idx: '0' })
```

Comment:
Shall the `idx` be a string? I feel like an integer would be easier. Also, shall we do this instead:
```js
  const addIdx = card => ({ idx: '0', ...card })
```
… so it doesn’t override it if defined?

Reply:
It was getting converted to a string anyways when passed to rwcDuplicates so my tests were failing (since I use === to check for idx equality). Since we never use it as an integer, I left it this way